### PR TITLE
[CHK-1249] feat(release): trigger automatic PR on eCommerce projects on commons release

### PR DESCRIPTION
**Pull request by giovanniberti**
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/create_pr_with_version_update.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+permissions:
+  pull-requests: write
+
+jobs:
+  create_pr:
+    strategy:
+      matrix:
+        repo:
+          - pagopa-ecommerce-transactions-service
+          - pagopa-ecommerce-scheduler-service
```
Comment (pietro-tota) [inline .github/workflows/create_pr_with_version_update.yaml:18]:
```suggestion
          - pagopa-ecommerce-transactions-service
          - pagopa-ecommerce-scheduler-service
          - pagopa-ecommerce-transactions-scheduler-service 
```

**Review by pietro-tota — COMMENTED** [review]
add batch to modules for which create automatic PR

File: .github/workflows/create_pr_with_version_update.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        uses: WyriHaximus/github-action-get-previous-tag@v1
+
+      - name: Replace old version with new version
+        uses: jacobtomlinson/gha-find-replace@v3
+        with:
+          find: "<pagopa-ecommerce-commons.version>\\d+\\.\\d+\\.\\d+</pagopa-ecommerce-commons.version>"
+          replace: "<pagopa-ecommerce-commons.version>${{ steps.latest_tag.outputs.tag }}</pagopa-ecommerce-commons.version>"
+          regex: true
+          include: "pom.xml"
+
```
Comment (pietro-tota) [inline .github/workflows/create_pr_with_version_update.yaml:40]: Duplicate this task in order to perform eCommerce commons version replacement also for helpdesk gradle build file. See https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/blob/main/build.gradle.kts#L23

Comment (giovanniberti) [reply .github/workflows/create_pr_with_version_update.yaml:39]: Fixed in efe9936

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**